deploy/roles/eod/tasks/main.yaml

```diff
@@ -245,3 +252,163 @@
+    - pixmap-miscfixed-font
+    - pixmap-fixed-fonts
+    - abyssinica
+    - tree 
+  when: custom_repo                             ####  ??????
+  tags:
+    - eod 
+
+- name: Remove gpk-update-icon file from /etc/xdg/autostart directory 
+  file:
+    path: /etc/xdg/autostart/gpk-update-icon.desktop
+    state: absent 
+  tags:
+    - eod 
+
+- name: Add /usr/local/eod/conf/admin/Xconfig/*.cfg files to register 
+  shell: ls -l /usr/local/eod/conf/admin/Xconfig 
+  register: cfg_files 
+  tags:
+    - eod 
+
+- debug: 
+    var: cfg_files
+
+- name: Add XkbExtension to Passive.cfg file 
+  lineinfile: 
+    dest: /usr/local/eod/conf/admin/Xconfig/Passive.cfg 
+    line: XkbExtension=1
+    insertafter: [Protocol]
+    state: present
+  when: cfg_files.stdout == "0"
+  tags:
+    - eod 
+
+- name: Add XkbExtension to Passive_WithShadowSetting.cfg file 
+  lineinfile:
+    dest: /usr/local/eod/conf/admin/Passive_WithShadowSetting.cfg 
+    line: XkbExtension=1
+    insertafter: [Protocol]
+    state: present 
+  when: cfg_files.stdout == "0"
+  tags:
+    - eod 
+
+- name: Add XkbExtension to XDMCP_Broadcast.cfg file 
+  lineinfile:
+    dest: /usr/local/eod/conf/admin/XDMCP_Broadcast.cfg
+    line: XkbExtension=1
+    insertafter: [Protocol]
+    state: present       
+  when: cfg_files.stdout == "0"
+  tags:                  
+    - eod  
+
+- name: Add XkbExtension to XDMCP_Query.cfg file 
+  lineinfile:
+    dest: /usr/local/eod/conf/admin/XDMCP_Query.cfg
+    line: XkbExtension=1
+    insertafter: [Protocol]
+    state: present       
+  when: cfg_files.stdout == "0"
+  tags:                  
+    - eod  
+
+- name: Add Composite to Passive.cfg file 
+  lineinfile:
+    dest: /usr/local/eod/conf/admin/Passive.cfg
+    line: Composite=1
+    insertafter: XkbExtension=1
+    state: present       
+  when: cfg_files.stdout == "0"
+  tags:                  
+    - eod  
+
+- name: Add Composite to Passive_WithShadowSetting.cfg file 
+  lineinfile:
+    dest: /usr/local/eod/conf/admin/Passive_WithShadowSetting.cfg
+    line: Composite=1
+    insertafter: XkbExtension=1
+    state: present                   
+  when: cfg_files.stdout == "0"
+  tags:                              
+    - eod
+
+- name: Add Composite to XDMCP_Broadcast.cfg file 
+  lineinfile:                             
+    dest: /usr/local/eod/conf/admin/XDMCP_Broadcast.cfg
+    line: Composite=1                     
+    insertafter: XkbExtension=1               
+    state: present                        
+  when: cfg_files.stdout == "0"
+  tags:                                   
+    - eod
+
+- name: Add Composite to XDMCP_Query.cfg file 
+  lineinfile:                           
+    dest: /usr/local/eod/conf/admin/XDMCP_Query.cfg
+    line: Composite=1                       
+    insertafter: XkbExtension=1                 
+    state: present                          
+  when: cfg_files.stdout == "0"
+  tags:                                     
+    - eod      
```

Comment:
@olivermel What's the significance of cfg_files.stdout == "0"?  Does this mean that if any of the files are found, it will run these scripts on all of them?